### ISSUE TITLE
Python version clarification

### ISSUE THREAD

**Issue by lanodan** [passff/passff-host#17]
I see that passff-host works with python3, but it seems like it works with python3.5 (https://github.com/passff/passff-host/pull/3) but what about later/greater versions like python3.6?

**Comment by 5bentz** [comment]
Reading your previous issue, I'd assume you are a package maintainer.

`passff-host` definitely works with Python 3.6 since Arch users are happy!

For more information, I'd say:
* https://docs.python.org/3/faq/general.html#how-does-the-python-version-numbering-scheme-work
* which is not really* https://semver.org/ 

*Of course, we cannot know for sure if Python 3 will or will not introduce breaking changes...
For example, Python 3.6 requires that all optional parameters of `json.load(..)` are keyword-only, which is a sort of incompatibility...
Following a warning, some functions are deprecated:(See sys.getcheckinterval()) `Deprecated since version 3.2: Use getswitchinterval() instead.`

However, we are not using obscure features of the *sys* library ;)
* `subprocess.run` is the new high-level API since Python 3.5 (the old one is not even deprecated)
* `json.loads and dumps` have had only a minor new feature
* `sys.std{in,err}.buffer.write and read` no comment.
* `struct` ...

#### In short
I'm not really knowledgeable in Python's versioning scheme, however the documentation is pristine clear about previous changes in the language: the breaking ones are scarce and seems to especially hit debug functions.

As a side note, PassFF should require Firefox 48 (not tested) which introduced the stable webext API ;) PassFF currently works fine with Firefox 52 ESR.
